kernel/build/config-amd64

```diff
@@ -4941,7 +4941,13 @@ CONFIG_XFS_POSIX_ACL=y
 # CONFIG_XFS_WARN is not set
 # CONFIG_XFS_DEBUG is not set
 # CONFIG_GFS2_FS is not set
-# CONFIG_BTRFS_FS is not set
+CONFIG_BTRFS_FS=m
+CONFIG_BTRFS_FS_POSIX_ACL=y
```

Comment:
is this required?

Reply:
The option does not affect the resulting kernel, the ACL support will be build into the module.